README.md

```diff
@@ -138,21 +178,28 @@ CREATE TABLE persons(
 Write Ruby code that will create an instance of a person...
 
 ```rb
-# Your code goes here...
+abc= Person.new(name:abc,age: 53)
```

Comment:
The value that comes after `name:` needs to be a string. You're referencing the variable `abc`, which at the moment is nothing.